spec/unit/provider/oneview_san_manager_spec.rb

```diff
@@ -15,108 +15,116 @@
 ################################################################################
 
```

Comment:
Would be good to add 'c7000' to the file name and the provider: 'c7000' to the resources for future updates.
The file could also be replicated for synergy.